test/recipes/75-test_quicapi_data/ssltraceref-zlib.txt

```diff
@@ -88,7 +88,7 @@ Received Datagram
   Length: 91
 Sent Frame: Crypto
     Offset: 0
-    Len: 263
+    Len: 269
```

Comment:
Footnote, this could easily be 3 additional (2-bytes each) signature algorithms in the TLS client Hello (ML-DSA-44, ML-DSA-65 and ML-DSA-87).